ow_testbed/launch/spawn.launch

```diff
@@ -25,6 +25,7 @@
   <!-- Convert joint states from Gazebo to tf-tree for rviz -->
   <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" >
     <param name="publish_frequency" value="30" />
+    <remap from="/joint_states" to="/faults2/joint_states" />

```

Comment:
What is '/faults2' for?

Reply:
I think that was another test function! I'll double check and remove those

Reply:
@lanseafood this is outdated .. we are commenting on a piece of code that has been already removed